### PR TITLE
create virtual environment in python

### PR DESCRIPTION
**Pull request by hema-prudvi**

### CODE REVIEW

**Comment by ytbryan** [comment]
Hi @hema-prudvi , the syntax for a link is []()

I think you are missing a [ ]

File: README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ Good luck.
 - Python
   - [Python Documentation](https://www.python.org/doc/)
   - [Flask](https://flask.palletsprojects.com/en/1.1.x/)
+   -How to Create Python Virtual Environments on Ubuntu 18.04(https://linuxize.com/post/how-to-create-python-virtual-environments-on-ubuntu-18-04/)
```
Comment (ytbryan) [inline README.md]: missing a [ ] 

should be 

```
   -[How to Create Python Virtual Environments on Ubuntu 18.04](https://linuxize.com/post/how-to-create-python-virtual-environments-on-ubuntu-18-04/)
```

File: README.md [hunk 2]
```diff
@@ -32,6 +32,7 @@ Good luck.
 - Python
   - [Python Documentation](https://www.python.org/doc/)
   - [Flask](https://flask.palletsprojects.com/en/1.1.x/)
+   -[How to Create Python Virtual Environments on Ubuntu 18.04](https://linuxize.com/post/how-to-create-python-virtual-environments-on-ubuntu-18-04/)
```
Comment (ytbryan) [inline README.md:35]: @hema-prudvi the indentation of the line 35 is not align with line 34

Comment (hema-prudvi) [reply README.md:35]: fixed indentation

Comment (huskypower) [reply README.md:35]: Thank you hema for fixing the indentation , and welcome more contributions from you

**Pull request merged**